spyder/utils/qthelpers.py

```diff
@@ -502,7 +502,42 @@ def show_std_icons():
     dialog.show()
     sys.exit(app.exec_())
 
-    
+
+def create_control_layout(controls, main_widget):

```

Comment:
Maybe it is better if we have 

`create_control_layout(widgets=None, layout=None, main_widget)` and we can either provide one or the other but not both?